docs/apache-airflow-providers-openlineage/guides/developer.rst

```diff
@@ -23,6 +23,42 @@
 Implementing OpenLineage in Operators
```

Comment:
General nit: this doc gets very long, maybe it should be split?

Reply:
I agree, it's not short, but I think it's still the best way of presenting this information instead of splitting it into separate documents. There are three main parts to this page: three methods of lineage extraction, plus some additional information like how to contribute, troubleshoot, etc. I believe it's important to keep this information together, but I am open to discussion. Nevertheless, I think even if we decide to split it, we can create another PR and merge this one for now.